### ISSUE TITLE
Pipe terminal out to window

### ISSUE THREAD

**Issue by ShimShamSam** [microsoft/vscode-remote-release#660]
When using the integrated terminal on my local machine, I can pipe output to VSCode using something along the lines of:

```bash
echo "foo" | code -
```

Is there a way to do this in Remote Development- SSH?

**Comment by chrmarti** [comment]
@aeschli Should this already work? I see 'Reading...' in the terminal, an editor tab opens, but the editor remains empty:
![image](https://user-images.githubusercontent.com/9205389/98341987-0592bb80-2010-11eb-9410-4b5537ef0d28.png)

**Comment by aeschli** [comment]
Yes, this should work. I just tested this with Remote-SSH and with a 'Try a sample' container and it worked.
The cli command comes back when the editor is closed.

**Comment by chrmarti** [comment]
![Nov-06-2020 14-20-28](https://user-images.githubusercontent.com/9205389/98370702-46ec9080-203b-11eb-8b47-c3602513f642.gif)

It doesn't immediately work in a new terminal. Opening and closing another editor makes the content appear. Maybe some buffer is not flushed?

**Comment by aeschli** [comment]
The content is written to a temp file while the editor is opening. Maybe file changes are not tracked properly by the editor?

I don't see the bug, so @chrmarti if you can debug it on you machine that would be great.
